# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-04-18 18:28+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: categories.py:79
msgid "All modules"
msgstr "Alle Module"

#: categories.py:80
msgid "Configuration and monitoring of system services"
msgstr ""

#: categories.py:81
msgid "System"
msgstr ""

#: categories.py:82
msgid "System overview"
msgstr ""

#: categories.py:83
msgid "Wizards"
msgstr ""

#: categories.py:84
msgid "Wizards for simple configuration"
msgstr ""

#: syntax.py:119
#, python-format
msgid "Unknown syntax %s"
msgstr "Unbekannte Syntax %s"

#: syntax.py:131
#, python-format
msgid ""
"Base type verification failed (type %(base)s): no verify function defined"
msgstr ""
"Basistypprüfung is fehlgeschlagen (Typ %(base)s): Es ist keine "
"Überprüfungsfunktion definiert"

#: syntax.py:136
#, python-format
msgid ""
"Base type verification failed (type %(base)s): function to verify syntax "
"could not be found (%(function)s)"
msgstr ""
"Basistypprüfung is fehlgeschlagen (Typ %(base)s): Es konnte keine "
"Überprüfungsfunktion gefunden werden"

#: verify.py:65
msgid "Value is not an boolean"
msgstr "Es ist kein bool'scher Wert"

#: verify.py:71
msgid "Value is not an integer"
msgstr "Es ist kein ganzzahliger Wert"

#: verify.py:77
#, fuzzy
msgid "Value is not n floating point number"
msgstr "Es ist keine Gleitkommazahl"

#: verify.py:85
#, python-format
msgid "Invalid regular expression (%(regex)s): %(message)s"
msgstr "Ungültiger regulärer Ausdruck (%(regex)s): %(message)s"

#: verify.py:91 verify.py:130
msgid "Value is not of type string"
msgstr "Der Wert ist keine Zeichenkette"

#: verify.py:96 verify.py:100
msgid "Value contains invalid characters"
msgstr "Im Wert sind ungültige Zeichen enthalten"

#: verify.py:106
msgid "Value is not a list"
msgstr "Es ist keine Liste"

#: verify.py:112
msgid "Value is not a dictionary"
msgstr "Es handelt sich nicht um ein Dictionary"

#: verify.py:133
msgid "No valid choices specified"
msgstr "Es wurden keine Auswahlmöglichkeiten definiert"

#: verify.py:136
msgid "Value does not match any of the specified choices"
msgstr "Der Wert entspricht keinem der angegebenen Auswahlmöglichkeiten"

#: protocol/definitions.py:55
msgid "OK, operation successful"
msgstr "OK, Operation erfolgreich"

#: protocol/definitions.py:56
msgid "OK, containing report message"
msgstr "OK, enthält Nachricht"

#: protocol/definitions.py:57
msgid "OK, partial response"
msgstr "OK, Partielle Antwort"

#: protocol/definitions.py:58
msgid "OK, operation successful ask for shutdown of connection"
msgstr "OK, Anfrage zum Beenden des Moduls"

#: protocol/definitions.py:59
msgid "Command currently not available/possible"
msgstr "Kommando momentan nicht ausführbar oder nicht verfügbar"

#: protocol/definitions.py:60
msgid "non-fatal error, processing may continue"
msgstr "Nicht fataler Fehler, Bearbeitung wird fortgesetzt"

#: protocol/definitions.py:61
msgid "Bad request"
msgstr "Fehlerhafte Anfrage"

#: protocol/definitions.py:62
msgid "Unauthorized"
msgstr "Nicht authorisiert"

#: protocol/definitions.py:63
msgid "Invalid command arguments"
msgstr "Ungültige Kommandoargumente"

#: protocol/definitions.py:64
msgid "Forbidden"
msgstr "Verboten"

#: protocol/definitions.py:65
msgid "Not found"
msgstr "Nicht gefunden"

#: protocol/definitions.py:66
msgid "Command not allowed"
msgstr "Kommando ist nicht erlaubt"

#: protocol/definitions.py:67 protocol/definitions.py:79
msgid "Unknown command"
msgstr "Unbekanntes Kommando"

#: protocol/definitions.py:68
msgid "Authentication failed"
msgstr "Authentisierung fehlgeschlagen"

#: protocol/definitions.py:69
msgid "Account is expired"
msgstr "Konto ist abgelaufen"

#: protocol/definitions.py:70
msgid "Account is disabled"
msgstr "Konto ist deaktiviert"

#: protocol/definitions.py:71
msgid "Access to console daemon is prohibited"
msgstr "Der Zugriif zum UMC-Server ist untersagt"

#: protocol/definitions.py:72
msgid "Command execution prohibited"
msgstr "Die Kommandoausführung ist untersagt"

#: protocol/definitions.py:73
msgid "Internal error"
msgstr "Interner Fehler"

#: protocol/definitions.py:74
msgid "Request could not be found"
msgstr "Die Anfrage konnte nicht gefunden werden"

#: protocol/definitions.py:75
msgid "Module process died unexpectedly"
msgstr "Modulprozess hat sich unerwartet beendet"

#: protocol/definitions.py:76
msgid "Connection to module process failed"
msgstr "Verbindung zum Modulprozess ist fehlgeschlagen"

#: protocol/definitions.py:77
msgid "SSL server certificate is not trustworthy"
msgstr "SSL-Zertifikat ist nicht vertrauenswürdig"

#: protocol/definitions.py:78
msgid "Unparsable message header"
msgstr "Unverständlicher nachrichtenkopf"

#: protocol/definitions.py:80
msgid "Invalid number of arguments"
msgstr "Ungültige Anzahl von Argumenten"

#: protocol/definitions.py:81
msgid "Unparsable message body"
msgstr "Unverständlicher nachrichtenrumpf"

#: protocol/definitions.py:82
msgid "Error occuried during command processing"
msgstr "Fehler ist während der Kommandoverarbeitung aufgetreten"

#: protocol/definitions.py:83
msgid "Specified locale is not available"
msgstr "Angegebene Sprache ist nicht verfügbar"

#: protocol/definitions.py:113
msgid "Unknown state"
msgstr "Unbekannter Status"

#: protocol/message.py:155
#, python-format
msgid "The command '%s' do not have any arguments"
msgstr "Das Kommando '%s' hat keine Argumente"

#: modules/__init__.py:84 modules/__init__.py:101
#, python-format
msgid ""
"Execution of command '%(command)s' has failed:\n"
"\n"
"%(text)s"
msgstr "i Die Ausführung des Kommandos %(command)s ist fehlgeschlagen:\n"
"\n"
"%(text)s"
